src/main/kotlin/org/rust/lang/core/RustFileElementType.kt

```diff
@@ -0,0 +1,12 @@
+package org.rust.lang.core
+
+import com.intellij.psi.stubs.IndexSink
+import com.intellij.psi.tree.IStubFileElementType
+import org.rust.lang.RustLanguage
+import org.rust.lang.core.stubs.RustFileStub
+
+object RustFileElementType : IStubFileElementType<RustFileStub>(RustLanguage) {
+    override fun indexStub(stub: RustFileStub, sink: IndexSink) {
+        super.indexStub(stub, sink)
```

Comment:
I'm not sure what's goinig on here, but if I omit this override I get

```
e: /home/matklad/projects/intellij-rust/src/main/kotlin/org/rust/lang/core/RustFileElementType.kt: (8, 1): 

Object 'RustFileElementType' must be declared abstract or implement abstract base class member 
public abstract fun indexStub(@org.jetbrains.annotations.NotNull p0: org.rust.lang.core.stubs.RustFileStub!, @org.jetbrains.annotations.NotNull p1: com.intellij.psi.stubs.IndexSink): kotlin.Unit 

defined in com.intellij.psi.tree.IStubFileElementType
```

But the function is **not** abstract (https://github.com/JetBrains/intellij-community/blob/master/platform/core-impl/src/com/intellij/psi/tree/IStubFileElementType.java#L65)